components/dashboard/src/teams/TeamUsage.tsx

```diff
@@ -241,30 +244,66 @@ function TeamUsage() {
                                         </ItemField>
                                     </Item>
                                     {currentPaginatedResults &&
-                                        currentPaginatedResults.map((usage) => (
-                                            <div
-                                                key={usage.instanceId}
-                                                className="flex p-3 grid grid-cols-5 justify-between transition ease-in-out rounded-xl focus:bg-gitpod-kumquat-light"
-                                            >
-                                                <div className="my-auto">
-                                                    <span>{getType(usage.workspaceType)}</span>
-                                                </div>
-                                                <div className="my-auto">
-                                                    <span className="text-gray-400">{usage.workspaceClass}</span>
-                                                </div>
-                                                <div className="my-auto">
-                                                    <span className="text-gray-700">{getMinutes(usage)}</span>
-                                                </div>
-                                                <div className="my-auto">
-                                                    <span className="text-gray-700">{usage.credits.toFixed(1)}</span>
-                                                </div>
-                                                <div className="my-auto">
-                                                    <span className="text-gray-400">
-                                                        {displayTime(usage.startTime)}
-                                                    </span>
+                                        currentPaginatedResults.map((usage) => {
+                                            return (
+                                                <div
+                                                    key={usage.instanceId}
+                                                    className="flex p-3 grid grid-cols-12 gap-x-3 justify-between transition ease-in-out rounded-xl focus:bg-gitpod-kumquat-light"
```

Comment:
**issue**: The kumquat classes inherited from the `ItemList` component can be removed, but let's keep this out of the scope of these changes here. Ideally, the table header should not use a different background color on hover.

```suggestion
                                                    className="flex p-3 grid grid-cols-12 gap-x-3 justify-between transition ease-in-out rounded-xl dark:bg-gray-800"
```